CONFIGURING-DETECTORS.md

```diff
@@ -0,0 +1,103 @@
+# Configuring Detectors for Edge Inference
+
+## Do I need to configure my detectors?
+
+No detector-specific configuration is necessary for basic use of the edge endpoint. Once it's running, submitting an image query to the edge endpoint will automatically trigger the creation of an edge inference pod for the specified detector. This inference pod will be created with the default settings. Once the pod has loaded the model and is ready to serve requests, the edge model will begin attempting to answer image queries sent to it. If its confidence is above the confidence threshold for the detector, you'll receive the answer from the edge model. If the confidence is too low, it will escalate the query to the cloud. 
+
+## Why would I want to configure detectors?
+
+Configuring detectors for the edge endpoint allows you to provide finer-grained control over their behavior on the edge. By configuring a detector, you can:
+* Have the edge endpoint automatically create an inference pod for a detector, ensuring it's ready to serve requests without having to first submit a query to it
+* Configure a detector to always return the edge model's answer, regardless of the confidence
+* Configure a detector to never escalate queries to the cloud
+* Specify a maximum frequency for cloud escalations
+
+## How do I configure a detector?
+
+Detector configurations are specified in [edge-config.yaml](configs/edge-config.yaml). The file has three sections: `global_config`, `edge_inference_configs`, and `detectors`. 
+
+NOTE: After modifying the config file, you'll have to re-run [setup-ee.sh](deploy/bin/setup-ee.sh) for your changes to be reflected.
+
+### `global_config`
+
+The global config contains parameters that affect the overall behavior of the edge endpoint. 
+
+#### `refresh_rate`
+
+`refresh_rate` is a float that defines how often the edge endpoint will attempt to fetch updated ML models (in seconds). If you expect a detector to frequently have a better model available, such as if you are labeling many image queries for a new detector, you can reduce this to ensure that the improved models will quickly be fetched and deployed. In practice, you likely won't want this to be lower than ~30 seconds due to the time it takes to train and fetch new models. If not specified, the default is 60 seconds.
+
+### `edge_inference_configs`
```

Comment:
could we name this something so it's clear it's at the detector level?

Reply:
Ah I see it can apply to multiple detectors. Hmmm

Reply:
Yeah the naming here is tricky, I spent a bit of time going back and forth about it. I'm definitely down to brainstorm some more options to try to make it more clear (though it's tough without making it super long!).